ecosystem/README.md

```diff
@@ -15,14 +15,11 @@
   Team. If enough the approval requirements are met by SEP team members, the SEP will move towards 
   `FCP`. Otherwise, it'll regress to a `Draft`.
 * **FCP** — A SEP that has entered a Final Comment Period (FCP). After one week has passed, during
-  which any new concerns should be addressed, the SEP's status will become `Active` or `Final`,
-  depending on the content of the SEP.
+  which any new concerns should be addressed, the SEP's status will become `Active`.
 * **Active** - An actively maintained SEP that is intended for immediate adoption by the entire
-  ecosystem by its author(s), and if it is a Standard, by SDF as well. Additional updates may be
-  made without changing the SEP number.
-* **Final** - A finalized SEP that is intended for immediate adoption by the entire
-  ecosystem by its author(s), and if it is a Standard, by SDF as well. A final SEP should only be
-  updated to correct errata.
+  ecosystem. Additional updates may be made without changing the SEP number.
+* **Final** - A finalized SEP that is being used in live products and will not be changed aside from
+  minor errata.
```

Comment:
This statement makes it sound like the primary reason for a proposal to be a candidate for being 'Final' is that it has been adopted in live products. I think we could have 'Active' proposals that have a few live implementations that aren't ready for being finalized.

I think we should reverse the sentence and make the focus that the primary attribute of a 'Final' proposal is that it will not change.

>A finalized SEP will not be changed aside from minor errata. For a proposal to be a candidate to be made `Final` it must be being used in live products.